utils/build/virtual_machine/provisions/host-auto-inject-install-script/auto-inject_host_script.yml

```diff
@@ -7,4 +7,7 @@
         local_path: utils/build/virtual_machine/provisions/auto-inject/tracer_debug/debug_config.yaml
   remote-command: |
     bash execute_install_script.sh host 
-    sudo cp debug_config.yaml /etc/datadog-agent/inject/debug_config.yaml
\ No newline at end of file
+    sudo mkdir -p /var/run/datadog-installer
+    sudo mkdir -p /opt/datadog/apm/inject/run
+    printf "DD_APM_RECEIVER_SOCKET=/opt/datadog/apm/inject/run/apm.socket\nDD_DOGSTATSD_SOCKET=/opt/datadog/apm/inject/run/dsd.socket\nDD_USE_DOGSTATSD=true\n" | sudo tee /var/run/datadog-installer/environment
+    sudo cp debug_config.yaml /etc/datadog-agent/inject/debug_config.yaml
```

Comment:
I think this cp is a mistake. We don't need this configuration for host scenarios (we are configuring the auto inject using the env variables).
We need this file for container scenarios.
I think it was my mistake, and it's here for long time ago. I didn't realize until it start failing

Reply:
Sounds good, I'll remove it

Reply:
Actually this looks required, without it the injection fails with 

```
{"level":"error","ts":1718881200.3642743,"caller":"process/main.go:187","msg":"error when parsing","error":"error loading config: open /etc/datadog-agent/inject/debug_config.yaml: no such file or directory","stacktrace":"main.main\n\t/go/src/github.com/DataDog/auto_inject/preload_go/cmd/process/main.go:187\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:271"}
```